README.md

```diff
@@ -1,2 +1,154 @@
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install
+
+### library we use that are not in python standard library
+$ pip install requests
+
+$ pip install -U click
+
+Let's start the installation process first you need to install library that we mention 
+then we will clone this github into your local computer
+
+ssh 
+> $ git clone git@github.com:ta-assistant/TA-CLI.git
+
+https 
+> $ git clone https://github.com/ta-assistant/TA-CLI.git
+
+and you need to goto TA-CLI directory to install CLI.
+
+
+Goto TA-CLI directory 
+```bash
+$ cd TA-CLI
+```
+install CLI ( for more infomation -> [CLI](https://github.com/ta-assistant/TA-CLI/tree/master/ta_cli#readme) )
+
+```bash
+$ pip install --editable .
+```
+
+Now you have completed the installation process.
+
+## Getting start
+#### Available Commands
+  1. init    [Init TA's work directory Args: workID (str): Work's ID]
+  2. login   [Login]
+  3. start   [Start working on TA directory]
+  4. submit  [Submit]
+
+#### Login
+first let login we login with apikey that you need to ask server manager to get it then assume that your apikey is `1234567890`
+then type command ta login --apikey 1234567890
+```bash
+$ ta login --apikey `yourapikey`
+```
+then you should get this massage
+```bash
+~\User\key has been created
+~\User\key\taconfig.json has been created.
+```
+now your apikey will kept in your User directory
+```bash
+~/user/key/taconfig
+```
+So now you have apikey that in your computer next if you want to check students work you must create new directory that have students work in it when you use our command you must use it in that directory path. In this example I will call it work_directory.
+#### Init work directory
+let's start. I will assume that you already have lot of students work files in your work directory. then type command (this command you need to have work id if you don't know what is work id -> [work_id])
+```bash
+$ ta init --workid `work id`
+```
+if everything successful, you will receive this message.
+```
+[*] ~\Users\dir\work_directory makeing work directory
+ |-[/] Creating workDirectory ~\Users\dir\work_directory
+ |-[/] Creating `config.json`
+ |-[/] Checking API-KEY
+ |-[/] fetching draft.json ...
+ |  |-[/]   API Request
+
+  * statusCode : 200
+  * message : Success
+  * requestId : *****************
+  * workDraft : {'outputDraft': ['studentId', 'param1', 'param2', 'comment', 'score', 'scoreTimestamp'], 'fileDraft': '{studentId}_test.zip'}
+
+ |
+[/] ~\Users\dir\work_directory is ready
+```
+if you don't know what workDraft for -> [work draft]
+
+now you will have
+```bash
+$ ./work_directory
+       ├── stuwork1
+       ├── stuwork2
+       ├── stuwork3
+             .
+             .
+             .
+       └── ta
+            ├──  draft.json
+            └── config.json
+             
+```
+#### Start checking work
+if everything is ready now you can use `ta start` ( if something is't ready ta start will send you what is missing )
```

Comment:
suggestion:
If everything is ready, now you can use `ta start` ( if something isn't ready `ta start` will send you what is missing )